### DISCUSSION TITLE
Playing streams from obs and from mobile diferent size on screen player

### DISCUSSION THREAD

**Discussion by nocpbxchap** [ant-media/Ant-Media-Server#6033] [General]
Hi, i stream from obs and from mobile my question is why on the playing part if publishing from obs the size of player is diferent if we play from mobile.

![mobile](https://github.com/ant-media/Ant-Media-Server/assets/82979248/d8701c0b-80db-411f-8df4-3f2a92881442)
![obs](https://github.com/ant-media/Ant-Media-Server/assets/82979248/efa96641-86a8-47fe-b6cf-633d9719fc8f)

these is thumbail but on player things also look diferent and i am using abr, but nothing changes, any way to both show same.

regards

**Comment by yashtandon113** [comment]
@nocpbxchap 

When you play a video with an aspect ratio of 16:9 (for example, a 1920x1080 stream) on a mobile screen in portrait mode, the video player adjusts the display to fit the screen. This change can result in black bars on the sides or top or bottom of the video, or the video may be zoomed in or cropped to fit the screen.

It is normal practice to guarantee that the video is displayed correctly on the mobile device's screen without distortion. If you change the device screen to landscape mode, it tries to fit according to the aspect ratio, and you will have the full screen covered.

Please let us know if you have any questions.

**Reply by nocpbxchap** [reply]
yes correct fixed with some css changes, now if you see example i am using thumbail feature of antmedia, here on that section how we control the size of the png image displayed as these section still show diferent if coming from obs or from mobile?
![image](https://github.com/ant-media/Ant-Media-Server/assets/82979248/8abdbe62-d29c-4ba8-9f69-b2fdd65e0ba8)

**Reply by yashtandon113** [reply]
@nocpbxchap 

When you publish a stream with OBS with a resolution like 1280x720, which is a 16:9 aspect ratio, but when you publish the WebRTC stream from mobile (in portrait mode) directly using a sample WebRTC page, the resolution changes, for example, to 480x640, which is a 3:4 aspect ratio.

The server keeps the same aspect ratio even during transcoding, so this is why you see different aspect ratios in thumbnails when you publish via mobile and OBS Studio.
